cache/jvm/src/main/scala/coursier/Cache.scala

```diff
@@ -796,10 +796,10 @@ object Cache {
         val file = localFile(url, cache, artifact.authentication.map(_.user))
 
         def res =
-          if (url.startsWith("file:/")) {
+          if (url.startsWith("local:/")) {
```

Comment:
`local:/` should probably be a constant somewhere in the core, in order to avoid a lot of duplications in the rest of the review as well.

nit: like `http://` being the protocol, `local://` (double /) would follow the convention better.

Reply:
Not necessarily… `file:/path/to/something` is valid (no "authority" part), and doesn't start with `file://`.